circle.yml

```diff
@@ -4,6 +4,7 @@ machine:
     GIT_SOURCE_REPO: https://github.com/git/git.git
     GIT_EARLIEST_SUPPORTED_VERSION: v2.0.0
     GIT_LATEST_SOURCE_BRANCH: master
+    GIT_NEXT_SOURCE_BRANCH: master
```

Comment:
@larsxschneider great catch! I fixed that up in: 
https://github.com/git-lfs/git-lfs/pull/2470/commits/a2e6e2d21cb904e09963715693fa64dc367c17e1.